test/docs.js

```diff
@@ -111,10 +111,17 @@ describe('documentation', function() {
       global: global
     };
 
+    function FakeExpress() {
+      return {
+        get: function() {}
```

Comment:
We don't really though, any code that runs inside of a callback isn't tested. That's been a known limitation of the docs testing since it was implemented. Just using this an example since I worked on it today-- everything starting here isn't tested: https://github.com/GoogleCloudPlatform/gcloud-node/blob/f52d1a8268af90b6132b03ada3bee19b5cd139a0/lib/storage/file.js#L1200

I've added your code, but this still doesn't save us... it just gets us one layer deeper in the execution path.